CONTRIBUTING.md

```diff
@@ -77,3 +79,53 @@ Add an entry to the changelog when your PR:
 #### In the [Readme](README.md)
 The readme is the first place a lot of people look for information about the repository. Update any parts of the readme that are affected by your PR.
 
+
+## Development Environment
+
+There are two ways to get live compiler feedback while developing Obelisk libraries. For libraries that are dependencies of the `skeleton` application, the easiest way to get feedback is to
+
+```bash
+cd skeleton
+ob run
+```
+
+This `ob run` session only loads modules that are dependencies of the skeleton and `obelisk-run`. For example, `obelisk-asset` and `obelisk-route` are used by the skeleton.
+
+For other libraries like `obelisk-command` you can use `ghcid`. To launch `ghcid` for `lib/command` you can run
+
+```bash
+nix-shell -A obeliskEnvs.obelisk-command --run "cd lib/command && ghcid"
```

Comment:
Is https://github.com/obsidiansystems/obelisk/blob/master/scripts/ghcid-this-lib recommended?

Reply:
This is simple enough that I prefer to just document it over writing magic scripts.